src/main/java/com/github/rnewson/couchdb/lucene/util/Analyzers.java

```diff
@@ -51,114 +60,199 @@
         public Analyzer newAnalyzer(final String args) {
             return new BrazilianAnalyzer();
         }
+		
+		@Override
+		public Analyzer newAnalyzer(final JSONObject args) {
+			return new BrazilianAnalyzer();
+		}
     },
     CHINESE {
         @Override
         public Analyzer newAnalyzer(final String args) {
             return new SmartChineseAnalyzer();
         }
+		
```

Comment:
same comment for this and all the ones below. this code didn't change, so it should not appear in the diff.